guidance_plugin_validator/config/params.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2021 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# Double: The time allocated for system startup. Validation begins after this much time has passed.
+# Units: s
+startup_duration: 60.0
```

Comment:
Not sure of the pros/cons of a fixed startup time, vs waiting for drivers ready from the health monitor

Reply:
I agree. I updated the implementation to wait for the 'DRIVERS_READY' message from the health monitor.